keyboards/butterstick/keymaps/default/keymap.c

```diff
@@ -0,0 +1,184 @@
+#include QMK_KEYBOARD_H
+
+#include "butterstick.h"
```

Comment:
you don't need this one
```suggestion

```

Reply:
Fixed in https://github.com/qmk/qmk_firmware/pull/5742/commits/ed263ea00aa379ffd144ad9219f682cddeaadda0